Untitled.ipynb

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Looks like this should be deleted or integrated into tests/Charts.ipynb.

Reply:
Done!